### DISCUSSION TITLE
need a help

### DISCUSSION THREAD

**Discussion by Gitcot** [community/community#65113] [Programming Help]
I want to find python languages community

**Comment by arafajs** [comment]
- Python Developers Community – LinkedIn
https://www.linkedin.com/groups/25827/

- Python Discord
https://pythondiscord.com/

- Python Reddit
https://www.reddit.com/r/Python/

**Reply by Gitcot** [reply]
thanks

**Comment by lab596** [comment]
Cool these are my top 3 suggestions: 

1. Search for Python Repositories:
Go to the GitHub website (https://github.com/) and use the search bar to search for repositories that are written in Python. You can simply type "language:python" in the search bar to filter repositories that are primarily written in Python.

2. Follow Contributors:
Check out the profiles of contributors to a Python repository. You can follow them to stay updated on their activity and discover other projects they might be involved in.

3. Explore Python Organizations:
Many active Python communities are organized under GitHub organizations. For example, the Python Software Foundation has an organization on GitHub with various repositories related to the Python programming language.

**Reply by Gitcot** [reply]
thank a lot

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Gitcot ! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
